UPGRADE.md

```diff
@@ -66,6 +66,11 @@ There is a list of all the repositories maintained by monorepo, changes in log b
 - [#530 - Update of installation for production via docker](https://github.com/shopsys/shopsys/pull/530)
     - update `docker-compose.yml` on production server with the new configuration from updated [`docker-compose.prod.yml`](./project-base/docker/conf/docker-compose.prod.yml.dist) file
     - update `nginx.conf` with configuration from updated [`nginx.conf`](./project-base/docker/nginx/nginx.conf)
+- [#547 - content-test directory is used instead of content during the tests](https://github.com/shopsys/shopsys/pull/547)
+    - modify your `parameters_test.yml.dist`, `parameters_test.yml`, `paths.yml` according to this pull request so there will be used different directory for feeds, images, etc., during the tests
+    - modify your `build-dev.xml`, add a new phing target `test-dirs-create` and add it as a dependency after each `dirs-create` in this file
+    - modify your `nginx.conf`, change location for images from `^/content/images/` to `^/content(-test)?/images/`
+    - modify your `routing_front.yml`, change configuration for routes `front_image`, `front_image_without_type` - replace `/content/` by `/%shopsys.content_dir_name%/`
```

Comment:
also, users need to modify their `paths.yml` - they have to replace hardcoded "content" with "%shopsys.content_dir_name%"

Reply:
Already mentioned - modify your `parameters_test.yml.dist`, `parameters_test.yml`, `paths.yml` according to this pull request - is this not enough?